site/content/2022-12-09.md

```diff
@@ -0,0 +1,119 @@
+Candidates must be nominated by a core team member. If the candidate is a core team member, they may nominate themselves.
+Nominees (in alphabetical order):
+- [Brett Cannon](https://discuss.python.org/t/steering-council-nomination-brett-cannon-2023-term/21078)
+- [Emily Morehouse](https://discuss.python.org/t/steering-council-nomination-emily-morehouse-2023-term/21329)
+- [Dong-hee Na](https://discuss.python.org/t/steering-council-nomination-dong-hee-na-2023-term/21424/1)
+- [Pablo Galindo Salgado](https://discuss.python.org/t/steering-council-nomination-pablo-galindo-salgado-2023-term/21307)
+- [Gregory P. Smith](https://discuss.python.org/t/steering-council-nomination-gregory-p-smith-2023-term/21332/1)
+- [Victor Stinner](https://discuss.python.org/t/steering-council-nomination-victor-stinner-2023-term/21407/1)
+- [Petr Viktorin](https://discuss.python.org/t/steering-council-nomination-petr-viktorin-2023-term/21478)
+- [Thomas Wouters](https://discuss.python.org/t/steering-council-nomination-thomas-wouters-2023-term/21147)
+
+<iframe width="560"
+                height="315"
+                src="https://www.youtube.com/embed/cbufmr-DJ3U"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+
+
+### [Security Patches and Updates to Python 3.11, 3.10, 3.9, 3.8, and 3.7](https://pythoninsider.blogspot.com/2022/12/python-3111-3109-3916-3816-3716-and.html)
+
+submitted by [kjaymiller](https://github.com/kjaymiller) on 2022-12-07T06:29:47Z
+
+This update is for 3.11.1, 3.10.9, 3.9.16, 3.8.16, 3.7.16 and the latest 3.12 alpha (3).
+Another all supported versions update usually means security patches and this one is no different. There are 4 security updates for all of the updated versions and 3 bonus security fixes for the older versions (3.7-3.10).
+3.11.1 also features more commits than 3.10.1 did from it's [dot-zero] release.
+> A whopping 495 new commits since 3.11.0. This is a massive increase of changes comparing to 3.10 at the same stage in the release cycle: there were “only” 339 commits between 3.10.0 and 3.10.1.
+Python 3.11 was release at the end of October and was praised for its massive performance updates and updates to error handling among many other features.
+
+<iframe width="560"
+                height="315"
+                src="https://www.youtube.com/embed/1e8_tjB07zs"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+
+
+
+
+
+
+
+## Conferences and Events
+
+
+### [PyCascades Tickets Available Now](https://pretix.eu/pycascades/vancouver-2023/)
+
+submitted by [kjaymiller](https://github.com/kjaymiller) on 2022-12-09T05:38:46Z
+
+
+_No response_
```

Comment:
```suggestion
```

Reply:
Yeah I need to update GH-Issues with a fix